rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -7719,6 +7719,125 @@ public void getEstimatedFeesForNextPegOutEvent_zero_pegouts() throws IOException
         assertEquals(Coin.ZERO, bridgeSupport.getEstimatedFeesForNextPegOutEvent());
     }
 
+    @Test
+    public void test_processPegoutsInBatch_before_RSKIP271_activation_processPegoutsIndividually() throws IOException {
```

Comment:
This test is not verifying what it states in its name. This could be achieved by adding a second pegout and after running the updateCollections you should have 2 elements in releaseTransactionSet

Reply:
Resolved